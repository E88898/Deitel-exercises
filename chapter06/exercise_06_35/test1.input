18
Y
32
35
N
