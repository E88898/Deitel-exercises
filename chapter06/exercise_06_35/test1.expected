How much is 3 times 6? Very good!

How much is 7 times 5? No. Please try again. Very good!

